patches/node/feat_initialize_asar_support.patch

```diff
@@ -35,20 +30,20 @@ index 250a43c5455b4f9ff72dd3a34d5b0aa270f43cc6..2e35cbba2ac02494c44821af395fe019
  }
  
 +function setupAsarSupport() {
-+  process._linkedBinding('electron_common_asar').initAsarSupport(require);
++  processLinkedBinding('electron_common_asar').initAsarSupport(require);
 +}
 +
  function patchProcessObject(expandArgv1) {
    const binding = internalBinding('process_methods');
    binding.patchProcessObject(process);
 diff --git a/test/fixtures/policy/process-binding/app.js b/test/fixtures/policy/process-binding/app.js
-index 16e26d12a160286b1b6aaeb64b15668b05b9865b..a287d0a2363acbf24077eec040116f96ef18a7b3 100644
+index 16e26d12a160286b1b6aaeb64b15668b05b9865b..93d5afe0bd0cd733620547932009dc1dc531841d 100644
 --- a/test/fixtures/policy/process-binding/app.js
 +++ b/test/fixtures/policy/process-binding/app.js
-@@ -5,6 +5,3 @@ const assert = require('assert');
- assert.throws(() => { process.binding(); }, {
+@@ -6,5 +6,5 @@ assert.throws(() => { process.binding(); }, {
    code: 'ERR_ACCESS_DENIED'
  });
--assert.throws(() => { process._linkedBinding(); }, {
+ assert.throws(() => { process._linkedBinding(); }, {
 -  code: 'ERR_ACCESS_DENIED'
--});
++ code: 'ERR_ACCESS_DENIED'
```

Comment:
This is the nittiest of nits :smile_cat: but I think the whitespace indentation is off by one space?